src/Essentials/src/MediaPicker/MediaPicker.android.cs

```diff
@@ -66,46 +66,74 @@ public async Task<FileResult> CaptureAsync(MediaPickerOptions options, bool phot
 			if (!OperatingSystem.IsAndroidVersionAtLeast(33))
 				await Permissions.EnsureGrantedAsync<Permissions.StorageWrite>();
 
-			var capturePhotoIntent = new Intent(photo ? MediaStore.ActionImageCapture : MediaStore.ActionVideoCapture);
+			var captureIntent = new Intent(photo ? MediaStore.ActionImageCapture : MediaStore.ActionVideoCapture);
 
-			if (!PlatformUtils.IsIntentSupported(capturePhotoIntent))
-				throw new FeatureNotSupportedException($"Either there was no camera on the device or '{capturePhotoIntent.Action}' was not added to the <queries> element in the app's manifest file. See more: https://developer.android.com/about/versions/11/privacy/package-visibility");
+			if (!PlatformUtils.IsIntentSupported(captureIntent))
+				throw new FeatureNotSupportedException($"Either there was no camera on the device or '{captureIntent.Action}' was not added to the <queries> element in the app's manifest file. See more: https://developer.android.com/about/versions/11/privacy/package-visibility");
 
-			capturePhotoIntent.AddFlags(ActivityFlags.GrantReadUriPermission);
-			capturePhotoIntent.AddFlags(ActivityFlags.GrantWriteUriPermission);
+			captureIntent.AddFlags(ActivityFlags.GrantReadUriPermission);
+			captureIntent.AddFlags(ActivityFlags.GrantWriteUriPermission);
 
 			try
 			{
 				var activity = ActivityStateManager.Default.GetCurrentActivity(true);
 
-				// Create the temporary file
-				var ext = photo
-					? FileExtensions.Jpg
-					: FileExtensions.Mp4;
```

Comment:
Is this technically a breaking change since we don't do the extension for videos anymore? From what I can tell in the sample app, now it looks more like a URL than a filename? If people are depending on this now weird things might happen?

Reply:
If you look at the new code paths, the filename we generate here is only used for photos, not videos, where the path is returned for us.  This _is_ different if you consider the code _before_ #19844 where we still were passing the output uri we expected the captured video to save to.  

I guess the question is, if capture video worked on some api levels on 8.0.3 then there might be a bit of a change in behaviour?  I'm not sure there's much we can do though if this is the more 'correct' way to fix this (by using the returned location/value for videos instead of asking them to use a specific uri)